<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Travelling Salesman Problem</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <widget class="QFrame" name="frmAlgo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>205</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Algorithm:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboAlgo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnRun">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>273</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>273</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Data Options</string>
   </property>
   <widget class="QRadioButton" name="optGenerate">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Generate a new TSP data set</string>
    </property>
    <property name="toolTipDuration">
     <number>3</number>
    </property>
    <property name="text">
     <string>Generate Data</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="optExisting">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Upload a data set from the TSP library</string>
    </property>
    <property name="toolTipDuration">
     <number>3</number>
    </property>
    <property name="text">
     <string>Upload Exisitng Data File</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frmResults">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lblExec">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>411</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblDistance">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wdGraph" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>511</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frmGenerate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>431</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLineEdit" name="txtVertices">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can only enter numbers in this field.</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblVertex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No of vertices</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtConnect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can only enter numbers in this field.</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMax">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can only enter numbers in this field.</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>You can only enter numbers in this field.</string>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>42</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connectivity</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMax">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>40</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max edge weight:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMin">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>12</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min edge weight:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSym">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>12</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Symmetric</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_FileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSaveAs">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse_Save">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>69</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnData">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>176</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QFrame" name="frmUpload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="lbl_FileName_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFileName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>15</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse_Open">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>14</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Format:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboFormat">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkHideGraph">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Do not show graph</string>
   </property>
  </widget>
  <zorder>frmResults</zorder>
  <zorder>frmAlgo</zorder>
  <zorder>btnRun</zorder>
  <zorder>btnClear</zorder>
  <zorder>grpData</zorder>
  <zorder>wdGraph</zorder>
  <zorder>frmGenerate</zorder>
  <zorder>btnData</zorder>
  <zorder>frmUpload</zorder>
  <zorder>chkHideGraph</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
